workloads/opea/chatqna/README.md

```diff
@@ -0,0 +1,91 @@
+# Deploy OPEA ChatQnA workload on OCP
+
+## Overview
+The workload is based on the [OPEA ChatQnA Application](https://github.com/opea-project/GenAIExamples/tree/v0.8/ChatQnA) running on Intel® Gaudi Accelerator with OpenShift and OpenShift AI. Refer to the [OPEA Generative AI Examples](https://github.com/opea-project/GenAIExamples/tree/v0.8) for more details about the OPEA workloads. 
+ 
+## Prerequisites
+* Provisioned RHOCP cluster. Follow steps [here](/README.md#provisioning-rhocp-cluster)
+* The Persistent storage using NFS is ready. Refer to [documentation](https://docs.openshift.com/container-platform/4.16/storage/persistent_storage/persistent-storage-nfs.html) for the details to set it up.
+
+**Note**: Refer to [documentation](https://docs.openshift.com/container-platform/4.16/storage/index.html) for setting up other types of persistent storages.
+* Provisioned Intel Gaudi accelerator on RHOCP cluster. Follow steps [../../../gaudi/README.md]
+* Installed and Configured RHOAI. Follow steps [here](../inference/README.md/#install-rhoai) 
+* Minio based S3 service ready for RHOAI. Follow steps [here](https://ai-on-openshift.io/tools-and-applications/minio/minio/#create-a-matching-data-connection-for-minio)
+* Create the Persistent Volume for the Application
+First, create ```oc create namespace opea-chatqna``` namespace.
+1. From CLI
+  
+**Note**: The NFS is used to provide the Persistent Volume as reference here. Refer to [Persistent storage documentation](https://docs.openshift.com/container-platform/4.16/storage/index.html) for other solutions
+
+Please update the NFS server IP and the path in the ```persistent_volumes.yaml``` file and deploy:
+
+``` 
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/workloads/opea/chatqna/persistent_volumes.yaml
+
+```
+``` 
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/workloads/opea/chatqna/persistent_volume_claims.yaml
+
```

Comment:
Can we merge the persistent_volumes_claims.yaml with chatqna_megaservice_deploy.yaml, and remove the steps to claim the persistent volumes?

Reply:
persistent volume claims take some time to bind to the volumes. and deployment is deployed at the same time. if the binding takes a while deployment might error as well. Hence the separate file